### ISSUE TITLE
sh not in path

### ISSUE THREAD

**Issue by legonerd605** [jkulvich/COTLTracker#7]
when I run the latest release for windows I get the error

```
PS C:\Users\legon\Desktop> ./player --test
exec: "sh": executable file not found in %PATH%
```

I have installed win-bash but what path name do I give it?

**Comment by jkulvich** [comment]
Yes, this is known trouble. Actually, this tool doesn't work with Windows :(
I use my linux and mac machine to develop. I will correct it for windows users in future updates.

As a temporary solution you can try to use WSL. But when I tried, I have got trouble with ADB forwarding.

Another way it's to use GIT Bash which adds support for sh and emulates unix-like environment.
